p2p/src/authenticated/discovery/config.rs

```diff
@@ -174,7 +186,9 @@ impl<C: Signer> Config<C> {
             max_handshake_age: Duration::from_secs(10),
             handshake_timeout: Duration::from_secs(5),
             allowed_connection_rate_per_peer: Quota::per_second(NZU32!(1)),
-            allowed_incoming_connection_rate: Quota::per_second(NZU32!(256)),
+            max_concurrent_handshakes: NZU32!(1_024),
+            allowed_handshake_rate_per_ip: Quota::per_second(NZU32!(16)), // 90 concurrent handshakes per IP
```

Comment:
Should this be 80? 16 handshakes per second * max 5 seconds per handshake

Reply:
😓